packages/expo-font/src/Font.ts

```diff
@@ -4,7 +4,7 @@ import { Platform } from '@unimodules/core';
 
 import ExpoFontLoader from './ExpoFontLoader';
 
-/**
+/*
```

Comment:
`/**` for doc blocks (unless we don't want this to show up as doc snippets in VS Code's TS plugin, etc)